flink-core/src/main/java/org/apache/flink/api/java/typeutils/runtime/TupleSerializer.java

```diff
@@ -76,7 +76,7 @@ public T createInstance() {
 	public T createInstance(Object[] fields) {
 
 		try {
-			T t = tupleClass.newInstance();
+			T t = tupleClass.getDeclaredConstructor().newInstance();
```

Comment:
this change may have performance implications as noted in the mail thread linked in the jira.